### ISSUE TITLE
Email people who are in the OLD Google Group about becoming members via Google Form

### ISSUE THREAD

**Issue by equivalentideas** [AOGPN/opengovernment.org.au#76]
This is because people who are currently in the Google Group haven't committed to the terms of reference etc. See #59 for the form and more details

People on this list could also review and give feedback on the terms.

**Comment by henare** [comment]
The new plan is to open the group up completely - #83 - so I think this can be closed.

**Comment by equivalentideas** [comment]
While the google group is going to be open ( see #83 ), as far as I understand things, the plan is still to have people agree to the terms of the network to become members of the Network (as opposed to members of the google group). Reopening this.

**Comment by henare** [comment]
I think this should come from the steering committee. I guess Peter will do it once everything is ready, does that sound right @kat?

**Comment by henare** [comment]
We should also probably say we're closing the old Group.

**Comment by kat** [comment]
again, one for Peter.

kat@oaf.org.au
Founder, Director, Worker OpenAustralia Foundation
http://openaustraliafoundation.org.au/
openaustralia http://openaustralia.org/ | planningalerts
http://planningalerts.org.au/| electionleaflets
http://electionleaflets.org.au/ | righttoknow http://righttoknow.org.au/
| theyvoteforyou http://theyvoteforyou.org.au/

**Comment by henare** [comment]
:+1:
